living_doc_utilities/model/issue.py

```diff
@@ -184,4 +222,4 @@ def is_valid_issue(self) -> bool:
 
         @return: True if the issue is valid, False otherwise.
         """
-        return all([self.repository_id is not None, self.title is not None, self.issue_number is not None])
+        return all([len(self.repository_id) > 0, len(self.title) > 0, self.issue_number > 0])
```

Comment:
Just a  suggestion, this should do the same thing:
```suggestion
        return all([self.repository_id, self.title, self.issue_number > 0])
```

Reply:
Proposal used. Thank you